assets/js/index.js

```diff
@@ -0,0 +1,112 @@
+// variables
+const weekDay = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+const startTime = 10;
+const endTime = 20;
+let times = [];
+// let todo = JSON.parse(localStorage.getItem("todoList")) || [];
+
+// get current date and time
+const getCurrentDate = () => {
+  // get current day
+  const day = moment().day();
+  const displayDay = weekDay[day];
+  // get current date
+  const date = moment().toString().split(" ");
+  const displayDate = date[2] + " " + date[1] + " " + date[3];
```

Comment:
Try to use template literal strings rather than string concatenation like you have here. Template literals are much easier to read, and less likely to lead to code errors. Here it would look like:

`const displayDate= `${date[2]} ${date[1]} ${date[3]}`